subsys/net/l2_wifi_if_conn/Kconfig

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2023 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+config L2_WIFI_CONN_WQ_STACK_SIZE
```

Comment:
Can you include a KConfig to enable/disable the connectivity integration? And have this KConfig rely on conn_mgr being enabled?

Reply:
Done